addons/account/views/account_move_views.xml

```diff
@@ -629,26 +635,28 @@
+                                           attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt'))]}"/>
                                     <span class="o_form_label mx-3 oe_edit_only"
-                                          attrs="{'invisible': [ '|', '|', ('state', '!=', 'draft'), ('invoice_payment_term_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"> or </span>
+                                          attrs="{'invisible': [ '|', '|', ('state', '!=', 'draft'), ('invoice_payment_term_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt'))]}"> or </span>
                                     <field name="invoice_date_due" force_save="1"
                                            placeholder="Date"
-                                           attrs="{'invisible': ['|', ('invoice_payment_term_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
+                                           attrs="{'invisible': ['|', ('invoice_payment_term_id', '!=', False), ('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt'))]}"/>
                                 </div>
-
-                                <field name="journal_id"
-                                       invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"
-                                       groups="account.group_account_readonly"
-                                       options="{'no_create': True}"
-                                       domain="[('type', '=?', invoice_filter_type_domain)]"
-                                       attrs="{'readonly': [('posted_before', '=', True)]}"/>
-
-                                <field name="company_id" groups="base.group_multi_company"/>
+                                <field name="invoice_date_due" force_save="1"
+                                       placeholder="Date"
+                                       attrs="{'invisible': [('move_type', 'not in', ('in_invoice', 'in_receipt', 'in_refund'))]}"/>
+                               
+
+                                <field name="invoice_payment_ref"
+                                       attrs="{'invisible':[('move_type', 'not in', ('in_invoice', 'out_invoice', 'in_refund'))], 'readonly': [('state', '=', 'paid'), ('move_type', '=', 'in_refund')]}"/>
+                                <field name="invoice_partner_bank_id"
+                                       context="{'default_partner_id': commercial_partner_id}"
+                                       domain="[('partner_id', '=', bank_partner_id)]"
+                                       attrs="{'invisible': [('move_type', 'not in', ('out_refund', 'in_invoice', 'in_refund'))], 'readonly': [('state', '=', 'paid')]}"/>
```

Comment:
state can't be paid, fix this by ('payment_state', 'in', ('paid', 'in_payment'))